### PR TITLE
Add possibility to send messages

### LINKED ISSUE #2: Create chat view

### PR DESCRIPTION
**Pull request by KoodiVille**
Includes an AI that answers to your messages!

Fix #2

### CODE REVIEW

File: src/chat/Chat.jsx [hunk 1]
```diff
@@ -43,14 +58,35 @@ const getDirection = (userId, classes) => {
...
+        setMessages(withResponse);
+    }
+
+    const print = (text) => {
+        if(text.length > 0) {
+            const newMessages = messages.concat({userId: '1', text: text});
+            setMessages(newMessages);
+            setTimeout(function () {
+                addDummyResponse(newMessages);
+           }, 1000);
```
Comment (iz4k) [inline src/chat/Chat.jsx]: The text input needs to be cleared after a successful submit

File: src/chat/Chat.jsx [hunk 2]
```diff
@@ -43,14 +58,35 @@ const getDirection = (userId, classes) => {
...
             <div className={classes.bubble}>
                 <div className={classes.button}>{message.text}</div>
             </div>
         </div>);
     });
-    return <div className={classes.container}>{chatBubbles}</div>
+    return( 
+        <div className={classes.container}>
+            <div>{chatBubbles}</div> 
+            <MessageInput callBack={print} />
```
Comment (iz4k) [inline src/chat/Chat.jsx]: image

Add more spacing to the input field top, also make the field span the full width of the page

File: src/chat/MessageInput.jsx [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+import { Input, IconButton, InputAdornment } from '@material-ui/core';
+import SendIcon from '@material-ui/icons/Send';
+
+const submit = (event, callBack, text) => {
+    if (event.key === 'Enter') {
+        event.preventDefault();
+        event.stopPropagation();
+       callBack(text);
+    }
+}
```
Comment (iz4k) [inline src/chat/MessageInput.jsx]: These should live inside the component and not in the global scope

Comment (iz4k) [inline src/chat/MessageInput.jsx]: Once the fn is inside the `MessageInput` component you can adjust it to be just `submit(event)` and if its enter do the preventdefaults and `props.callBack(text)` (text is in state and callback is in props now so no need to use as param in the submit)

File: src/chat/Chat.jsx (hunk 1 above)
Comment (KoodiVille) [reply src/chat/Chat.jsx]: I fixed this in the child components state.

File: src/chat/Chat.jsx (hunk 2 above)
Comment (KoodiVille) [reply src/chat/Chat.jsx]: I fixed this in the child components styles

**Pull request merged**